pages/docs/reference/compatibility.md

```diff
@@ -0,0 +1,114 @@
+- **BCB** - Full **B**ackward **C**ompatibility for Binaries
+  - Binaries (ABI)
+    - runtime: NV-binaries can be used  everywhere where OV binaries worked
+    - NV compiler: code compilable against OV binaries is compilable against NV binaries
+    - OV compiler may not accept NV binaries (e.g. those that exhibit newer language features or APIs)
+- **BC** = BCL & BCB
+- **EXP** - Experimental feature
+  - see below
+- **NO** - No compatibility guarantees
+  - we'll do our best to offer smooth migration, but can give no guarantees
+  - migration is planned individually for every incompatible subsystem 
+
+---
+
+\* No changes *modulo bugs* means that if an important bug is found (e.g. in the compiler diagnostics or elsewhere), a fix for it may introduce a breaking change, but we are always very careful with such changes.
+
+## Compatibility guarantees for Kotlin releases
+
+**Kotlin for JVM**:
+  - patch version updates (e.g. 1.1.X) are fully compatible
+  - minor version updates (e.g. 1.X) are backwards compatible
+
+| JVM  | 1.0 | 1.0.X | 1.1 | 1.1.X | ... | 2.0 |
+|---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| **1.0** | - | C | BC | BC | ... | ? |
+| **1.0.X** | C | - | BC | BC | ... | ?
+| **1.1** | BC | BC | - | C | ... | ?
+| **1.1.X** | BC | BC | C | - | .... | ?
+| **...** | ... | ... | ... | ... | ... | ... |
+| **2.0** | ? | ? | ? | ? | ... | - 
+
+**Kotlin for JS**: starting with Kotlin 1.1, both patch and minor version updates provide backward compatibility for the language (BCL), but no BCB.  
+
+| JS  | 1.0.X | 1.1 | 1.1.X | ... | 2.0 |
+|---:|:---:|:---:|:---:|:---:|:---:|
+| **1.0.X** | - |  EXP | EXP | ... | EXP |
+| **1.1** | EXP |  - | BCL | ... | ?
+| **1.1.X** | EXP | BCL | - | .... | ?
+| **...** | ... | ... | ... | ... | ... | ... |
+| **2.0** | EXP | ? | ? | ? | ... | - 
+
+**Kotlin Scripts**: both patch and minor version updates provide backward compatibility for the language (BCL), but no BCB.
+
+## Compatibility across platforms
+ 
+Kotlin is available for several platforms (JVM/Android, JavaScript and the upcoming native platforms). Every platform has its own peculiarities (e.g. JavaScript has no proper integers), so we have to adapt the language accordingly. Our goal is not to provide reasonable code portability without sacrificing too much.
+  
+Every platform may feature specific language extensions (such as platform types for JVM and dynamic types for JavaScript) or restrictions (e.g. some overloading-related restrictions on  the JVM), but the core language remains the same. 
+
+The Standard Library provides a core API available on all platforms, and we strive to make these APIs work in the same way on every platform. Along with these, the Standard Library provides platform-specific extensions (e.g. `java.io` for JVM, or `js()` for JavaScript), plus some APIs that can be called uniformly, but work differently (such as regular expressions for JVM and JavaScript).    
```

Comment:
Thanks, but I think it's clear enough as it stands, and would like to avoid repetition